src/main/java/unicash/ui/TransactionCard.java

```diff
@@ -112,42 +133,41 @@ public Label getTransactionLocation() {
     }
 
     /**
-     * Returns the categories label of the transaction card.
+     * Returns the first Category label of the transaction card.
      *
      * @return A Label containing the categories associated with the transaction.
      */
-    public Label getCategories() {
-        return this.categories;
+    public Label getFirstCategories() {
+        return this.firstCategory;
     }
 
 
-    /*
+    /**
      * For better presentation of the transaction amounts, instead of using the
      * toString method of the transaction amount directly, a dollar symbol is prepended
      * to the transaction, along with the polarity of the transaction depending on whether
      * it is an income or expense type, Accordingly, text fill styles are also used to
      * set the color of the amount Label text based on their type.
      */
     private void amountStyleFormatter() {
-        String transactionAmountWithDecimal = Amount.amountToDecimalString(transaction.getAmount());
-        String dollarTransactionWithDecimal = "$" + transactionAmountWithDecimal;
+        String formattedTransactionAmount = transaction.getAmount().toString();
+        String transactionType = transaction.getTypeString();
 
-        String transactionType = transaction.getType().toString().toLowerCase();
-        String transactionPolarity = transactionType.equals("expense") ? "-" : "+";
-        String transactionCardString = transactionPolarity + dollarTransactionWithDecimal;
+        String transactionSign = transactionType.equals("expense") ? "-" : "+";
```

Comment:
Can we compare the type by using the actual enumeration? Similar reason to a comment made earlier about providing a single source of truth